todo.md

```diff
@@ -23,3 +23,5 @@
     - config-message.component.spec.ts
     - config-group-title.component.spec.ts
     - config-previous-next-buttons.component.spec.ts
+
+7. Update import of ConfigUtilsService in config-previous-next-buttons.components.ts once ConfigUtilsService has been moved to lib.
```

Comment:
I think you don't need to mention this one. When you move the from storefrontlib to the product lib, you'll get compilation errors, which can serve as a reminder that you need to fix imports.

Reply:
Ok. I'll remove it.